### PR TITLE
[Landcover Toolkit] Add js extension to imports

### PR DESCRIPTION
**Pull request by gino-m**

### CODE REVIEW

File: toolkits/landcover/impl/bands.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * @fileoverview Static functions for band transformations.
  */
 
- var NamedArgs = require('users/google/toolkits:landcover/impl/named-args').NamedArgs;
+ var NamedArgs = require('users/google/toolkits:landcover/impl/named-args.js').NamedArgs;
```
Comment (gorelick-google) [inline toolkits/landcover/impl/bands.js]: extra leading whitespace here.

Comment (gino-m) [reply toolkits/landcover/impl/bands.js]: Are you referring to this?
image

If so it's actually part of GitHub's diff gutter (this has tricked me in the past as well).

**Comment by gino-m** [comment]
> I don't like required scripts needing the trailing .js, but the repo is weird if you look at it/use it outside of the code editor, so I think its the only option.

It seems some environments require this (or some config):
https://stackoverflow.com/questions/45673200/why-require-need-js-extension-while-importing-js-file

Not sure, but I think I needed it so that the code would work both in the Code Editor and in Node.js.

File: toolkits/landcover/impl/bands.js (hunk 1 above)
Comment (gorelick-google) [reply toolkits/landcover/impl/bands.js]: Yes, I'm referring to this line, but it's not github.  There's an extra leading space on this line.
Compare this to the next one:
image

image

Comment (gino-m) [reply toolkits/landcover/impl/bands.js]: Wow, good eye. Thanks!

**Comment by gino-m** [comment]
Fixed. Please restamp!

**Pull request merged**